File: asp/asp/Models/INVOICE.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace asp.Models
{
    public class INVOICE
    {
        [Key]
        public string INVOICE_ID { get; set; }
        public string INVOICE_CODE { get; set; }
        public string USER_ID { get; set; }
        public string RECEIVER { get; set; }
        public string RECEIVER_ADDRESS { get; set; }
        public string RECEIVER_PHONE { get; set; }
        public decimal TOTAL { get; set; }
        public string STATUS { get; set; }
        public string CREATE_ID { get; set; }
        public DateTime? CREATE_DATE { get; set; }
        [NotMapped]
        public List<INVOICE_DT> LST_INVOICE_DT { get; set; }
        [NotMapped]
        public string XML_INVOICE_DT { get; set; }
        public string INVOICE_STATUS_NAME { get; set; }
        public string METHOD_PAY { get; set; }
        public string CREATE_NAME { get; set; }
        public string USER_NAME { get; set; }
    }
}

File: 5_17_HeThongQuanLyVaBanLaptopHDSHOP/src/webadmin/src/app/core/models/INVOICE.ts
import * as moment from 'moment';
export class INVOICE {
  invoicE_ID: string | undefined;
  invoicE_CODE: string | undefined;
  useR_ID: string | undefined;
  receiver: string | undefined;
  receiveR_ADDRESS: string | undefined;
  receiveR_PHONE: string | undefined;
  total: number | undefined;
  status: string | undefined;
  creatE_ID: string | undefined;
  creatE_DATE: moment.Moment | undefined;
  invoicE_STATUS_NAME: string | undefined;
  creatE_NAME: string | undefined;
  useR_NAME: string | undefined;
  top: number | undefined;
}
